### ISSUE TITLE
Version 3.3.0 macOS issues

### ISSUE THREAD

**Issue by rpatters1** [frescobaldi/frescobaldi#1806]
Version 3.3.0

For me, the arm64 version crashes immediately upon launch:

<img width="372" alt="Screen Shot 2024-10-30 at 11 43 17 AM" src="https://github.com/user-attachments/assets/469ad074-8c08-44ea-aeee-c5f335f2d921">

The 86_64 version launches and seems to work, but if I quit and restart Frescobaldi, the main menu does not work correctly. On has to double-click the top menu names to see the the menu drop down, and the submenus are unreachable. The only fix I have found is to restart my mac.

Mac Studio M1 Max (2022)
macOS 12.7.6

**Comment by rpatters1** [comment]
P.S. So far I am not seeing the menu issue with 3.1.3, so I will stick with that for the time being.

**Comment by rpatters1** [comment]
P.P.S. I think the menu problem may be related to the fact that it opens without a document window. If I open it with a document window, it seems to work.

**Comment by SimonSeeberger** [comment]
Yes, this is a bug I see pretty often. You have to force Frescobaldi to close (sometimes even 2 times). After that it will open without a project loaded. I believe opening files in the startup process could be the issue here. I'm using arm64 MacOS, too.

**Comment by fedelibre** [comment]
Since version 4.0.0 macOS app is built using a new tool called Briefcase.

This issue can be closed.
